Library/Tracking/KSRAnalytics.swift

```diff
@@ -1648,7 +1645,7 @@ private func discoveryProperties(
     .withAllValuesFrom(parentCategoryProps ?? [:])
 
   result["everything"] = result.isEmpty
-  result["sort"] = params.sort?.rawValue
+  result["sort"] = params.sort?.trackingString
```

Comment:
Hm why did this change?

Reply:
I realized the sort values that we were sending to segment was using the rawValue which wasn't correct. The ticket requires the string values held by the trackingString property. It was until I tested for other enum cases that I found out

Reply:
good catch